### PR TITLE
Add RococoBridgeHub <> WococoBridgeHub full 2 way bridge

### PR DESCRIPTION
**Pull request by serban300**
Part of: #1662

### CODE REVIEW

File: relays/bin-substrate/src/chains/rococo_parachains_to_bridge_hub_wococo.rs [hunk 1]
```diff
@@ -76,3 +76,10 @@ impl CliBridgeBase for BridgeHubRococoToBridgeHubWococoCliBridge {
 	type Source = relay_bridge_hub_rococo_client::BridgeHubRococo;
 	type Target = relay_bridge_hub_wococo_client::BridgeHubWococo;
 }
+
+impl MessagesCliBridge for BridgeHubRococoToBridgeHubWococoCliBridge {
+	const ESTIMATE_MESSAGE_FEE_METHOD: &'static str =
```
Comment (svyatonik) [inline relays/bin-substrate/src/chains/rococo_parachains_to_bridge_hub_wococo.rs:81]: (not related to the PR) It should be removed in #1638 (sorry - there are still some traces of the old code).

File: relays/bin-substrate/src/cli/relay_headers_and_messages/parachain_to_parachain.rs [hunk 2]
```diff
@@ -118,7 +118,7 @@ macro_rules! declare_parachain_to_parachain_bridge_schema {
 				left_parachains_to_right_sign_override: [<$left_chain ParachainsTo $right_parachain SigningParams>],
 			}
 
-			impl [<$left_chain $right_parachain HeadersAndMessages>] {
+			impl [<$left_parachain $right_parachain HeadersAndMessages>] {
```
Comment (svyatonik) [inline relays/bin-substrate/src/cli/relay_headers_and_messages/parachain_to_parachain.rs:121]: Thank you! :)

**Review by svyatonik — APPROVED** [review]
LGTM! Thank you!

**Review by bkontur — APPROVED** [review]
cool, lgtm :)

**Pull request merged**